MapleServer2/Database/Classes/DatabaseAccount.cs

```diff
@@ -0,0 +1,125 @@
+﻿using Maple2Storage.Types;
+using MapleServer2.Enums;
+using MapleServer2.Types;
+using Newtonsoft.Json;
+using SqlKata.Execution;
+
+namespace MapleServer2.Database.Classes
+{
+    public static class DatabaseAccount
```

Comment:
Rather than calling this class `DatabaseAccount`, can it instead be called `Accounts` and not static, then in a file called `Database` create an instance of `Accounts` then expose it using a getter? So the ultimate caller uses something like `Database.getAccounts().Create()` or `Database.getAccounts().FindById()`